### ISSUE TITLE
E0308 warning popup doesn't display <> signs properly

### ISSUE THREAD

**Issue by lancelote** [intellij-rust/intellij-rust#5598]
## Environment

* **IntelliJ Rust plugin version:** 0.2.125.SNAPSHOT-202-dev
* **Rust toolchain version:** 1.44.0-nightly (b2e36e6c2 2020-04-22) x86_64-apple-darwin
* **IDE name and version:** CLion 2020.2 EAP (CL-202.5428.21)
* **Operating system:** macOS 10.15.4

## Problem description

Found the problem while checking #5548. `<` and `>` sings are displayed as `&#60;` and `&#62;`.

<img width="1015" alt="Screen Shot 2020-06-17 at 20 39 05" src="https://user-images.githubusercontent.com/2500320/84930895-a40e9680-b0da-11ea-9d6f-bd36d0171708.png">

## Steps to reproduce

```rust
use std::collections::HashMap;

fn fib_dyn(n: u64, map: &mut HashMap<u64, u64>) -> u64 {
    match n {
        0 | 1 => 1,
        n => {
            if map.contains_key(&n) {
                *map.get(&n).unwrap()
            } else {
                let val = fib_dyn(n - 1, map) + fib_dyn(n - 2, map);
                map.insert(n, val);
                val
            }
        }
    }
}

#[cfg(test)]
mod tests {
    use super::*;

    #[test]
    fn fib_5() {
        let mut map = HashMap::new();
        assert_eq!(fib_dyn(5, &map), 8);
    }
}
```

**Comment by vlad20012** [comment]
Oh, looks like a 2020.2 effect

**Comment by Kobzol** [comment]
So is this a platform bug? Or do we need to escape the text in a different way in 2020.2?

**Comment by Undin** [comment]
@Kobzol it's a combination of our fix of the corresponding bug in platform (< 2020.2) and the corresponding platform fix on 2020.2
Fix is on the way
